stix_shifter_modules/elastic_ecs/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -1910,14 +1910,42 @@
         }
       }
     },
-    "ip": {
-      "key": "x-oca-asset.ip",
-      "object": "observer"
-    },
-    "mac": {
-      "key": "x-oca-asset.mac",
-      "object": "observer"
-    },
+    "ip": [
+      {
+        "key": "ipv4-addr.value",
+        "object": "observer_ip"
+      },
+      {
+        "key": "ipv6-addr.value",
+        "object": "observer_ipv6"
```

Comment:
need to make sure if the value is ipv4 or ipv6 before setting the value. one IP can't be both ipv4 and v6. you may have to use a transformer to verify the value. i know few connectors has this confusions. but we are planning to fix them eventually.

Reply:
fixed. I added the transformer for ipv4 and v6.